backends/ebpf/CMakeLists.txt

```diff
@@ -94,6 +98,38 @@ set (EBPF_TEST_SUITES
   "${P4C_SOURCE_DIR}/testdata/p4_16_samples/*_ebpf.p4"
   )
 
-p4c_add_tests("ebpf-kernel" ${EBPF_DRIVER_KERNEL} ${EBPF_TEST_SUITES} "${XFAIL_TESTS_KERNEL}")
+set (SUPPORTS_KERNEL True)
+
+# Check if the kernel version is new enough to support ebpf features
+set (MIN_KERNEL 4.16.0)
+string (REGEX MATCH "[0-9]+[^-]*" KERNEL_VER ${CMAKE_SYSTEM})
+if (${KERNEL_VER} VERSION_LESS ${MIN_KERNEL} )
+  MESSAGE(WARNING "Kernel version too small, ignoring ebpf kernel tests...")
```

Comment:
Maybe you can output the minimum version required, so people know what to install